prime-angular-frontend/src/app/modules/enrolment/pages/overview/overview.component.ts

```diff
@@ -86,9 +91,14 @@ export class OverviewComponent extends BaseEnrolmentPage implements OnInit {
       message: 'When your enrolment is submitted for adjudication, it can no longer be updated. Are you ready to submit your enrolment?',
       actionText: 'Submit Enrolment'
     };
+
     this.busy = this.dialog.open(ConfirmDialogComponent, { data })
       .afterClosed()
       .pipe(
+        tap(() => {
+          this.ngBusyService.updateMessage(NG_BUSY_SUBMISSION_MESSAGE);
```

Comment:
In enrolment, currently not seeing any messages displayed on the screen.

Reply:
Hmm I just submitted an enrolment and seems to work fine.

If you're testing locally you may not see it as the request will complete before the message is displayed (I added a 5seconds delay before the message shows up)
![image](https://user-images.githubusercontent.com/34261366/152451010-01fbf518-3c74-4e41-aaee-525c788f93c8.png)

Reply:
![image](https://user-images.githubusercontent.com/59669288/152454927-97a80925-c004-430e-9d90-1c637865293e.png)

Reply:
Does the text only show up after 5 sec?

Reply:
Yes

Reply:
Yeah i would stick with immediately